code/game/machinery/civilian_bounties.dm

```diff
@@ -290,13 +290,24 @@
 		. += "<span class='notice'>Scan this in the cargo shuttle with an export scanner to register your bank account for the <b>[bounty_value * handler_tip]</b> credit handling tip.</span>"
 
 /obj/item/bounty_cube/process(delta_time)
-	if(COOLDOWN_FINISHED(src, next_nag_time))
-		if(!is_centcom_level(z) && !is_reserved_level(z)) //don't send message if we're on Centcom or in transit
-			radio.talk_into(src, "Unsent in [get_area(src)].[speed_bonus ? " Speedy delivery bonus of [bounty_value * speed_bonus] credit\s lost." : ""]", RADIO_CHANNEL_SUPPLY)
-			speed_bonus = 0
-			bounty_holder_account.bank_card_talk("\The [src] is unsent in <b>[get_area(src)]</b>.")
-			if(bounty_handler_account)
-				bounty_handler_account.bank_card_talk("\The [src] is unsent in <b>[get_area(src)]</b>.")
+	//if our nag cooldown has finished and we aren't on Centcom or in transit, then nag
+	if(COOLDOWN_FINISHED(src, next_nag_time) && !is_centcom_level(z) && !is_reserved_level(z))
+		//set up our nag message
+		var/nag_message = "\The [src] is unsent in [get_area(src)]."
+
+		//nag on Supply channel and reduce the speed bonus multiplier to nothing
+		var/speed_bonus_lost = "[speed_bonus ? " Speedy delivery bonus of [bounty_value * speed_bonus] credit\s lost." : ""]"
+		radio.talk_into(src, "[nag_message][speed_bonus_lost]", RADIO_CHANNEL_SUPPLY)
+		speed_bonus = 0
+
+		//alert the holder
+		bounty_holder_account.bank_card_talk("[nag_message]")
```

Comment:
Looks like this is duplicate code of the below, except without the null check...?

Reply:
You mean line 308? This alerts the bounty holder, i.e. the person who completed the bounty. That will never be null. The one with the null check (line 308) alerts the handler, i.e. the person who scanned the cube on the cargo shuttle with an export scanner (which sends a message to the person who completed the bounty, updating them).

Reply:
You sure?

![image](https://user-images.githubusercontent.com/35135081/107871653-73911f00-6e58-11eb-93a6-873d7a49c405.png)

Reply:
Oh you're right, those are two different words. Fair enough.